src/Mod/Sketcher/Gui/DrawSketchHandlerRotate.h

```diff
@@ -508,7 +508,7 @@ void DSHRotateController::configureToolWidget()
 
     toolWidget->setParameterLabel(
         WParameter::First,
-        QApplication::translate("TaskSketcherTool_p4_rotate", "Copies 'U'/'J'"));
+        QApplication::translate("TaskSketcherTool_p4_rotate", "Copies (modify with 'U'/'J')"));

```

Comment:
I'm concerned this string makes the whole box too wide. What about :
Sides (U/J to +/-)
or
Sides (U+,  J-)

Reply:
what about: Sides (+'U' / -'J')
Same syntax is used in other tool settings in sketcher, e.g. 'M'

Reply:
seems ok
Or we just remove this and put it in the tooltip.